### ISSUE TITLE
online training

### ISSUE THREAD

**Issue by InstantWindy** [kmaninis/OSVOS-PyTorch#23]
Hello! Is online training using the actual annotated image of the first frame of the training set as training data? Still using the first frame of the test set? I don't know what is online training. Thank you !

**Comment by scaelles** [comment]
Hello,
By online training, we mean that we use the first frame (mask and image) of the test sequence to learn the appearance of the object that we want to segment through the rest of the video sequence.
